package.json

```diff
@@ -51,44 +51,49 @@
     "check-dev": "npm run lint && npm run test-dev"
   },
   "dependencies": {
-    "babel": "^5.5.8",
-    "babel-core": "^5.5.8",
+    "babel": "^5.8.21",
+    "babel-core": "^5.8.22",
     "babel-loader": "^5.3.2",
+    "css-loader": "^0.20.1",
```

Comment:
This adds `css-loader` back to the Victory boilerplate, which I assume was removed due to Radium. IMO we should just keep it around.